synfig-studio/plugins/lottie-exporter/properties/offsetKeyframe.py

```diff
@@ -63,7 +63,7 @@ def clamped_tangent(p1, p2, p3, animated, i):
                 bias = 0.0
             tangent = (p2 - p1) * (1.0 + bias) / 2.0 + (p3 - p2) * (1.0 - bias) / 2.0
     elif p1 > p2:
-        if p2 >= p1 or p2 <= p3:
+        if p2 <= p3:
```

Comment:
So I recommend to change this in another PR, pointing out the mistake.

Reply:
Okay, I will do that.

Reply:
I suggest returning this line of code back. Here is the reason:
LGTM.com says that `p2> = p1` has no effect, because you already checked `elif p1> p2`.
But since there is an error in the first condition, you do not need to delete this condition, since it will become correct when the first condition changes.

Reply:
@ice0 Done. :smile: 
I have fixed that mistake in a different PR(#1233).